recipes/wham/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
This looks like it's in a separate directory than the recipe specified in the `meta.yaml`

Reply:
Sorry, I have to change the name of the file since there is another repo which has the same name and I got some files messed up.